src/chapters/typescript/studio.rst

```diff
@@ -0,0 +1,171 @@
+Studio: TypeScript
+==================
+
+Let's practice TypeScript by creating classes for Rocket cargo calculations.
+
+Requirements
+------------
+#. Fork `the starter repl.it <https://repl.it/@launchcode/rocket-studio-starter>`_
+#. Create classes for Astronaut, Cargo, and, Rocket  (Details below)
+
+   * All classes should be defined in their own files
+
+#. Use new classes to run a simulation in ``index.ts`` file
+
+In the starter you will notice that an ``interface`` named ``Payload`` has been declared.
+This interface insures that any class that implements it will have a ``weightKg`` property.
+Weight is a key property in this studio.
+
+Classes
+-------
+Define each of these classes in a separate files. Each class should be exported using ``export``.
+Example ``export class Astronaut {...``.
```

Comment:
Let's reformat these examples to make better sense. That includes the one on line 25.

Reply:
changed example layout. lmk if it looks better.